File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int n = nums.length;
        int cnt = 0;
        for(int i=1;i<n;i++)
        {
            if(nums[i]<nums[i-1]) cnt++;
        }
        if(nums[0]<nums[n-1]) cnt++;
        return cnt<=1;
    }
}

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.go
func check(nums []int) bool {
    cnt := 0
    n := len(nums)
    for i:=0;i<n;i++ {
        if nums[i] > nums[(i+1) % n] {
            cnt++
        }
        if cnt>1 {
            return false
        }
    }
    return true
}